assets/src/edit-story/app/font/fontProvider.js

```diff
@@ -53,13 +68,46 @@ function FontProvider({ children }) {
 
   const maybeEnqueueFontStyle = useLoadFontFiles();
 
+  const getMenuFonts = useCallback((fontFamilyList) => {
+    const newFontList = fontFamilyList.filter(
+      (fontName) =>
+        loadedFontFamily.current.findIndex((name) => name === fontName) < 0
+    );
+    if (!newFontList?.length) {
+      return new Promise((resolve) => resolve(''));
+    }
+    return fetch(
+      `${GOOGLE_MENU_FONT_URL}?family=${encodeURI(
+        newFontList.join('|')
+      )}&subset=menu`
+    )
+      .then((response) => response.body)
+      .then((body) => {
+        return body
+          .getReader()
+          .read()
+          .then(({ value }) => {
+            const decoder = new TextDecoder('utf-8');
```

Comment:
Let's just use `response.text()` instead of `body`.

Reply:
Done!